### PR TITLE
August 11 _lastUpdated Update

### PR DESCRIPTION
**Pull request by kbratti-va**
### Process

- [ ] Tests added or updated for change
- [ ] Docs updated
  - [ ] If you have made structural changes to the site, the [Developer Portal Content Types](https://community.max.gov/x/qUwOg) docs have been updated
  - [ ] If you have made/confirmed any decisions about user interactions and accessibility, the [Accessible Component Design](https://community.max.gov/x/xS8mgg) docs have been updated
- [ ] Accessibility concerns have been considered and addressed
  
  - [ ] If you've made any UI changes, you've run an axe check using the axe browser extension that reported no new issues

### CODE REVIEW

**Review by SlakrHakr — APPROVED** [review]
LGTM once UX approves

File: src/content/apiDocs/health/fhirAPIReleaseNotes.mdx [hunk 1]
```diff
@@ -1,3 +1,9 @@
+### August 11, 2022
+We have enabled an additional FHIR _sort parameter, _lastUpdated, for the following R4 resources:
+ - Medication
+ - Device
+
```
Comment (joeabraun) [inline src/content/apiDocs/health/fhirAPIReleaseNotes.mdx:5]: I would also just include this paragraph that we had in the previous two release notes, for consistency.
```suggestion
The _lastUpdated sort parameter lets you specify the arrangement of responses in ascending or descending order based on the last time they were changed. This parameter can be used alone or in conjunction with other sort parameters listed in the [Patient Health API (FHIR) OpenAPI specification](https://api.va.gov/internal/docs/fhir-r4/v0/openapi.json) to customize the order of responses to best meet your use case.
```

**Review by joeabraun — CHANGES_REQUESTED** [review]
Besides adding the extra paragraph, it looks good!

**Review by joeabraun — APPROVED** [review]
UX approved!

**Review by SlakrHakr — APPROVED** [review]
LGTM

**Pull request merged**